doc/classes/OS.xml

```diff
@@ -1058,7 +1070,7 @@
 		<member name="screen_orientation" type="int" setter="set_screen_orientation" getter="get_screen_orientation" enum="_OS.ScreenOrientation" default="0">
 			The current screen orientation.
 		</member>
-		<member name="tablet_driver" type="String" setter="set_current_tablet_driver" getter="get_current_tablet_driver" default="&quot;&quot;">
+		<member name="tablet_driver" type="String" setter="set_current_tablet_driver" getter="get_current_tablet_driver" default="&quot;wintab&quot;">
```

Comment:
That's a quirk of `doctool` which sometimes have platform-specific behavior, but you should revert this change manually.

Reply:
Will do tomorrow :)